tests/onnxruntime/test_modeling.py

```diff
@@ -240,6 +240,14 @@ def test_load_model_cuda_provider(self):
         self.assertListEqual(model.model.get_providers(), model.providers)
         self.assertEqual(model.device, torch.device("cuda:0"))
 
+    @require_torch_gpu
+    @pytest.mark.amdgpu_test
+    def test_load_model_rocm_provider(self):
+        model = ORTModel.from_pretrained(self.ONNX_MODEL_ID, provider="ROCMExecutionProvider")
+        self.assertListEqual(model.providers, ["ROCMExecutionProvider", "CPUExecutionProvider"])
+        self.assertListEqual(model.model.get_providers(), model.providers)
+        self.assertEqual(model.device, torch.device("cuda:0"))
```

Comment:
Will this test fail if the install of ORT + ROCM EP is not done correctly? If so, can we add a `@require_ort_rocm` or something similar? If not, can we make these new tests `@slow`? The CI is already huge.

Reply:
I don't think these tests should run by default as the marker of  `@pytest.mark.amdgpu_test` is present?

Reply:
No, this marker is useful only to select a subset of tests to run (`pytest -m "amdgpu_test"`).

If you try: `pytest tests/onnxruntime -k "test_load_model_rocm_provider" -s -vvvvv` this test is IMO likely to run while it should probably not unless ROCM EP is installed.

Reply:
Yes, the above line would run. What I meant was in our CI, the cpu tests wont't run it as it would require GPU?
and for the gpu tests only the `gpu_test` marker is selected. https://github.com/huggingface/optimum/blob/main/tests/onnxruntime/docker/Dockerfile_onnxruntime_gpu#L26

So these tests are not running in the CI? Am I missing something or anyother place the test might run?

But if you want that if a user/ developer is running locally, something like `@require_ort_rocm ` is nice?

Reply:
Ok will make the change with @require_ort_rocm . Could change the gpu_test, to `cuda_and_trt_ep_tests` as cuda and try ep are tested in same method .

Reply:
On second thought, creating two markers for existing gpu tests cuda_ep_test or trt_ep_test. As only some of them have art